packages/faceted-search/src/components/Badges/BadgeMenu/BadgeMenu.component.js

```diff
@@ -0,0 +1,95 @@
+import { useMemo } from 'react';
+import { isEmpty } from 'lodash';
+import PropTypes from 'prop-types';
+import Badge from '@talend/react-components/lib/Badge';
+
+import { BadgeFaceted } from '../BadgeFaceted';
+import { BadgeMenuForm } from './BadgeMenuForm.component';
+import { operatorPropTypes, operatorsPropTypes } from '../../facetedSearch.propTypes';
+
+const getSelectBadgeLabel = (value, t) => {
+	const labelAll = t('FACETED_SEARCH_VALUE_ALL', { defaultValue: 'All' });
+	if (!isEmpty(value)) {
+		return value.label;
+	}
+	return labelAll;
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const BadgeMenu = ({
+	id,
```

Comment:
is there a reason to not export this component as default? and not disable eslint rule

Reply:
actually no reason. I just follow how other badges are exported. for example [BadgeCheckboxes](https://github.com/Talend/ui/blob/master/packages/faceted-search/src/components/Badges/BadgeCheckboxes/BadgeCheckboxes.component.js#L25)

Reply:
I will merge this PR first since I have another task depending on this.
I will check in frontend channel to see if we can change to default exports, if we can, I will create another PR to change all.